### PR TITLE
HL-259 CTA and CTA List docs

### PR DESCRIPTION
**Pull request by ZofiaZ**

### CODE REVIEW

File: docs/guides/authoring/Howlite/cta/index.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# CTA & CTAs List Components
+
+## Usage
+CTA component renders a single link (An anchor html tag) with look & feel of a button or text link (controlled by style variants).
```
Comment (kamil-kwarciak) [inline docs/guides/authoring/Howlite/cta/index.md:4]: An anchor -> an anchor (make lower case)

Comment (kamil-kwarciak) [reply docs/guides/authoring/Howlite/cta/index.md:4]: (controlled by style variants) -> not sure if clear for regular reader

File: docs/guides/authoring/Howlite/cta/index.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+If you need just a single CTA, you can add it directly to a page section or container. To render multiple CTAs in one row, use **CTAs List** component and add buttons there.
+
+## Authorable properties
+Click "Edit" icon to see all CTAs component's properties:
+
+- You can provide a **Text** (label) for the CTA
+- add an **URL**. It can be either an external link starting with http(s) or an internal link. Type **/** to enable auto suggestions of internal pages.
+- optionally, you can selelect if the link should be **open in a new tab**, and if it should have an additional **Icon**.
```
[the remark this answers is not in the document]
Comment (kamil-kwarciak) [reply docs/guides/authoring/Howlite/cta/index.md:12]: could you check the spelling with Grammarly, etc?

[the remark this answers is not in the document]
Comment (ZofiaZ) [reply docs/guides/authoring/Howlite/cta/index.md]: done

File: docs/guides/authoring/Howlite/cta/index.md (hunk 1 above)
Comment (ZofiaZ) [reply docs/guides/authoring/Howlite/cta/index.md]: lower case letter - changed. For style variants, I made it a link to CTA variants section

**Comment by ZofiaZ** [comment]
merged, after slack approval from Kamil

**Pull request merged**